### ISSUE TITLE
[BUG]: README.md: Section "Install ble-testsuite" is no longer valid

### ISSUE THREAD

**Issue by gmacario** [B-AROL-O/ble-testsuite#30]
### Description of the bug

The instructions reported on the README.md are no longer valid:

```text
gmacario@hw0943:~$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/gmacario/ble-testsuite/HEAD/install.sh)"
curl: (22) The requested URL returned error: 404
gmacario@hw0943:~$
```

### Steps To Reproduce

1. Go to <https://ble-testsuite.netlify.app/>
2. Execute the command in section "Install ble-testsuite"

### Additional Information

IMHO this is an excellent issue for [Hacktoberfest 2023](https://hacktoberfest.com/).

Here are a few hints to have the PR accepted:

1. Carefully look at line
https://github.com/B-AROL-O/ble-testsuite/blob/main/README.md?plain=1#L11
and update it to match what the [online site](https://ble-testsuite.netlify.app/) says
2. Go to https://github.com/B-AROL-O/ble-testsuite/pulls and click "New pull request"
3. Fill in all the requested data, then click "Create"

**Comment by Chetax** [comment]
@gmacario i want to work on this issue please assign it to me

**Comment by gmacario** [comment]
> @gmacario i want to work on this issue please assign it to me

You are welcome @Chetax!

**Comment by pitdagosti** [comment]
Is this issue still open @gmacario?

**Comment by gmacario** [comment]
> Is this issue still open @gmacario?

No, it was fixed by #35. Thanks for noticing. 

Closing
